src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -159,12 +159,34 @@ public static Tag parseTag(String tag) throws ParseException {
     public static LocalDateTime parseAppointmentDateTime(String dateTime) throws ParseException {
         requireNonNull(dateTime);
         String trimmedDateTime = dateTime.trim();
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy HH:mm");
+        String[] dateTimeInfo = trimmedDateTime.split(" ");
+
+        if (dateTimeInfo.length != 2) {
+            throw new ParseException("Please ensure both Appointment date and time are entered in "
+                    + "dd-MM-yyyy HH:mm format.");
+        }
+
+        String dateInput = dateTimeInfo[0];
+        String timeInput = dateTimeInfo[1];
+        DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("dd-MM-yyyy");
+        DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HH:mm");
+
+        LocalDate date;
+        LocalTime time;
+
         try {
```

Comment:
Not sure if we should put these checks in separate methods (one for date, one for time) but maybe can consider

Reply:
I think it is good as in this way we do not need to change the `appointment` class constructor.